### PR TITLE
Add is_runahead task indicator.

### PR DESCRIPTION
**Pull request by hjoliver**
cylc-ui sibling of https://github.com/cylc/cylc-flow/pull/4103 (merge the cylc-flow PR first)

**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).
- [x] Already covered by (modified) existing tests.
- [x] Appropriate change log entry included.
- [x] Added new item the UI Quick Guide. No other documentation update required.

### CODE REVIEW

**Comment by hjoliver** [comment]
```ini
[scheduling]
  cycling mode = integer
  initial cycle point = 1
  [[queues]]
     [[[default]]]
        limit = 2
  [[graph]]
     P1 = foo
[runtime]
  [[foo]]
     script = "sleep 60"
```

shot

**Comment by hjoliver** [comment]
shot

**Comment by hjoliver** [comment]
I've gone for a simple dot as a task icon modifier. Obviously there is scope for something better, such as a small running man, or an arrow hitting a wall.

File: src/views/Guide.vue [hunk 1]
```diff
@@ -132,9 +132,24 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
...
+                  </v-list-item-icon>
+                  <v-list-item-content>
+                    <v-list-item-title>
+                      Runahead
+                    </v-list-item-title>
+                    <v-list-item-sub-title>
+                      Task held back by runahead limiting
+                    </v-list-item-sub-title>
+                  </v-list-item-content>
+              </v-list-item>
```
Comment (kinow) [inline src/views/Guide.vue:152]: 👍 I would have not remembered to update the Guide, even though I reviewed a PR for it this morning 🤣

**Review by kinow — APPROVED** [review]

**Comment by kinow** [comment]
p.s. I'm chasing that brittle functional test, not related to this change

**Comment by hjoliver** [comment]
Rebased.

**Review by kinow — APPROVED** [review]

**Comment by hjoliver** [comment]
The cylc-flow PR for this is now merged.

**Comment by hjoliver** [comment]
I'm going to merge this PR with one approval. It a small change that just replicates in cylc-ui what we already have in `cylc tui`, and apparently @oliver-sanders is taking leave this week.

**Comment by hjoliver** [comment]
(I've only updated the change log since tests passed, so I won't bother waiting for that).

**Pull request merged**